packages/opencascade/src/index.ts

```diff
@@ -1,10 +1,42 @@
-import initOpenCascade from 'opencascade.js';
-
-import opencascade from './jupytercad.opencascade';
+import opencascade, { OpenCascadeInstance } from './jupytercad.opencascade';
 import opencascadeWasm from './jupytercad.opencascade.wasm';
 
 export * as OCC from './jupytercad.opencascade';
 
+const initOpenCascade = (options: {
+  mainJS: any;
+  mainWasm: any;
+  worker?: any;
+  libs?: any[];
+  module?: any[];
+}): Promise<OpenCascadeInstance> => {
+  return new Promise((resolve, reject) => {
+    const { mainJS, mainWasm, worker, libs = [], module = [] } = options;
+    new mainJS({
+      locateFile(path) {
+        if (path.endsWith('.wasm')) {
+          return mainWasm;
+        }
+        if (path.endsWith('.worker.js') && !!worker) {
+          return worker;
+        }
+        return path;
+      },
+      ...module
+    }).then(async oc => {
+      for (const lib of libs) {
+        await oc.loadDynamicLibrary(lib, {
```

Comment:
Out of curiosity, what is this and why is it needed?

Reply:
I had an issue with building the Salome extension because of missing interfaces in `opencascade.js` so I tried to remove the `opencascade.js` dependency since we only need the init function. 
It turned out the problem was a missing typescript config in the Salome extension, we can roll back to using the function provided by `opencascade.js`